Gemfile.lock

```diff
@@ -63,6 +71,8 @@ GEM
       railties (>= 4.1.0, < 6.0)
       responders
       warden (~> 1.2.3)
+    domain_name (0.5.20180417)
+      unf (>= 0.0.5, < 1.0.0)
     dotenv (2.5.0)
     dotenv-rails (2.5.0)
```

Comment:
Should we specify this as:  gem 'dotenv-rails', groups: [:development, :test] ?

Reply:
Sure let's do it

Reply:
oh wait, actually it's already group there:
<img width="430" alt="screen shot 2018-08-13 at 6 52 41 pm" src="https://user-images.githubusercontent.com/13835666/44045849-264a298a-9f2a-11e8-9004-e70abc31acbe.png">